sycl/test/fpga_tests/fpga_aocx.cpp

```diff
@@ -21,3 +21,6 @@
 //
 // RUN: env SYCL_DEVICE_TYPE=ACC %t_aocx_src.out
 // RUN: env SYCL_DEVICE_TYPE=ACC %t_aocx_obj.out
+//
+// Clean up the archive file to avoid appending objects in the next test runs
+// RUN: rm %t_image.a
```

Comment:
I have 2 questions here:
 - why the clang cannot overwrite existing file? Can we let clang overwrite file if it exists?
 - if it is expected behavior I would prefer to remove the file before test execution (it can be created by some user manual actions).

Reply:
> Why the clang cannot overwrite existing file? Can we let clang overwrite file if it exists?

The archive-managing tool is not actually clang, but rather `llvm-ar` that merely gets called by the clang driver. The appending behavior is required by some of the FPGA usecases (e.g. adding the host object to the already-existing archive).

> If it is expected behavior I would prefer to remove the file before test execution (the file can be created by some user manual actions)

No problem, I believe a mere `touch` in the beginning would do the trick.

Reply:
Done. I've also tweaked the commit message.